packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -58,6 +58,13 @@
           "minimum": 500,
           "maximum": 120000,
           "description": "%connection_timeout_ms_description%"
+        },
+        "salesforcedx-vscode-apex-debugger.setBreakpointTimeoutMs": {
+          "type": "number",
+          "default": 10000,
+          "minimum": 500,
+          "maximum": 120000,
+          "description": "%set_breakpoint_timeout_ms_description%"
```

Comment:
Set-breakpoint timeout setting.

Reply:
Hopefully people won't need to set this one. I'm actually less inclined with exposing this for now since this actually is a technical detail that we have. If we expose this, then we need to support this even if we come up with a better, faster scheme in the future for setting breakpoints.

Could we hide this setting? Or do you want to have this setting exposed?

Reply:
That makes sense to me. I don't think there's a config property to hide it, so I'll just remove it for now. At least we know how to make this setting configurable.